README.md

```diff
@@ -0,0 +1,39 @@
+# Icinga DB
+
+1. [About](#about)
+2. [Documentation](#documentation)
+3. [Installation](#installation)
+4. [Configuration](#configuration)
+
+## <a id="About"></a>
+
+The Icinga DB web module is the way to go, to interact with the new database and to use the benefits it brings.
+
+This module reimplements and aims to replace the old monitoring module.
+It keeps all functionality from its predecessor and adds new quality of life features, such as:
+- bulk operations: set entire host groups into downtime with one operation
+- modals instead of tabs to speed up the workflow
+- a view switcher, that enables the user to choose how much detail should be displayed
+- the status bar now checks for set filters
+
+On top it offers a modern and streamlined design to give an even clearer and more concise overview over the most important points in your environment.
+
+
+![ListTypes](doc/res/ListTypes.jpg "Different levels of list details")
+
+![HostServiceList](doc/res/ScreenHostServiceList.jpg "Host and Service list")
+
+![HostDetailHistory](doc/res/ScreenHostDetailHistory.jpg "Host deatil and history")
+
+## <a id="Documentation"></a>Documentation
+
+The full documentation is located in the [doc/](doc/) directory
```

Comment:
Remove the word `full`, it is redundant.

Also, this link won't work on icinga.com/docs. Better link to `https://icinga.com/docs/<futureprojectname>`

Reply:
Removed the `full`
Not sure about the project name - so I'll leave this open for now :)